content/dstu2/encounter.md

```diff
@@ -7,18 +7,31 @@ title: Encounter | DSTU 2 API
 * TOC
 {:toc}
 
+## Terminology Bindings
+
+<%= terminology_table(:encounter, :dstu2) %>
+
+### Contained Location Bindings
+
+<%= terminology_table(:location, :dstu2) %>
```

Comment:
I'm a little hesitant to use :location as the symbol here. I think I would prefer something like :encounter_contained_location. 

My terminology_table logic isn't that smart and just assumes that the symbol matches a yaml definition file. This could be surprising when we eventually implement and document a real Location resource and suddenly the terminology table on this page gets a lot bigger and describes stuff that will never be returned in the contained version.